packages/lesswrong/components/recentDiscussion/RecentDiscussionTagRevisionItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { forumTypeSetting } from "../../lib/instanceSettings"
+import { Components, registerComponent } from "../../lib/vulcan-lib"
+
+const isEAForum = forumTypeSetting.get() === 'EAForum'
+// Pablo, Leo
+const megaTagUsers = ['BkbwT5TzSj4aRxJMN', 'pkJTc4xXhsCbNqkZM']
+const onlyStyleEditors = ['pkJTc4xXhsCbNqkZM']
```

Comment:
I would previously have hated this, but hear me out:
* This is only applicable to the EA Forum
* Their userIds are the same in all of {dev, staging, prod}

I'm just really not sold on the effort of making this a config variable.

Reply:
Seems reasonable to me. Of course, I'm a heathen.

Reply:
![image](https://user-images.githubusercontent.com/221511/150027669-f72e0727-991c-4dc6-9c62-2285d3bbdec6.png)

I guess it's fine

Reply:
> "metaTagUsers" is a confusing variable name though. I don't know what it's supposed to mean.

`megaTagUsers` with a g